enterprise/jakartaee11.platform/external/generated-jakarta.jakartaee-api-11.0.0-javadoc-license.txt

```diff
@@ -0,0 +1,93 @@
+Name: JakartaEE API 11.0.0 Documentation
+Version: 11.0.0
+License: EPL-v20
+Description: JakartaEE API 11.0.0 Documentation
+Origin: Generated from jakarta.jakartaee-api-10.0.0-M1-javadoc.jar
```

Comment:
Should be version 11:
`jakarta.jakartaee-api-10.0.0-M1-javadoc.jar`

Reply:
oups, thanks